packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -340,6 +341,14 @@ const Autocomplete = React.forwardRef(function Autocomplete(props, ref) {
     }
   }
 
+  if (startAdornment) {
+    const more = startAdornment.length - filterMax;
+    if (filterMax && !focused && more > 0) {
+      startAdornment = startAdornment.splice(0, filterMax);
+      startAdornment.push(<span key={filterMax} data-testid="more">{` + ${more} more`}</span>);
```

Comment:
We will need to localize this wording

Reply:
I don't get it. Can you add more context please?